HLTrigger/Configuration/python/HLT_75e33/eventsetup/hltOnlineBeamSpotESProducer_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+hltOnlineBeamSpotESProducer = cms.ESProducer("OnlineBeamSpotESProducer",
+    timeThreshold = cms.int32( 48 ),
```

Comment:
Note to self: technically, for Run-3 MC we use `1e6` because of [this](https://github.com/cms-sw/cmssw/blob/fa01ae41a33c9ecaa9771e2259c455a6e942f068/HLTrigger/Configuration/python/customizeHLTforCMSSW.py#L201-L211), but I understood the value does not matter in MC wfs, because there is a single IOV. Probably better to leave the more meaningful `48` as in the Run-3 menus (pre customisations).